### ISSUE TITLE
Failing build

### ISSUE THREAD

**Issue by tombishop1** [tombishop1/itraxBook#5]
Failing build:

```
label: constrained_clusters_1
Quitting from lines 1211-1228 (itraxBook.Rmd) 
Error in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  : 
  polygon edge not found
Calls: <Anonymous> ... <Anonymous> -> widthDetails -> widthDetails.text -> grid.Call
In addition: There were 12 warnings (use warnings() to see them)

Execution halted
Error: Process completed with exit code 1.
```

**Comment by tombishop1** [comment]
This doesn't fail build on Windows, so I wonder if its a macOS thing?

**Comment by tombishop1** [comment]
It was because of the latest version of Mac-os. Solved, for now, with a00307c
